src/content/learn/react-compiler.md

```diff
@@ -126,29 +126,46 @@ In addition to these docs, we recommend checking the [React Compiler Working Gro
 
 ### Installing eslint-plugin-react-compiler {/*installing-eslint-plugin-react-compiler*/}
 
-React Compiler also powers an eslint plugin. The eslint plugin can be used **independently** of the compiler, meaning you can use the eslint plugin even if you don't use the compiler.
+React Compiler also powers an ESLint plugin. The ESLint plugin can be used **independently** of the compiler, meaning you can use the ESLint plugin even if you don't use the compiler.
 
 <TerminalBlock>
 npm install -D eslint-plugin-react-compiler@beta
 </TerminalBlock>
 
-Then, add it to your eslint config:
+Then, add it to your ESLint config:
+
+```js
+import reactCompiler from 'eslint-plugin-react-compiler'
+
+export default [
+  {
+    plugins: {
+      'react-compiler': reactCompiler,
+    },
+    rules: {
+      'react-compiler/react-compiler': 'error',
+    },
+  },
+]
```

Comment:
in case this would be an accepted change, I actually tend to prefer:

1. a separate `configs` variable
2. usage of [the ESLint `Linter.Config[]` TS type](https://github.com/eslint/eslint/blob/d474443109762f3b92811df0411965cf64f595c2/lib/types/index.d.ts#L1402-L1406) with a JSDoc annotation

```suggestion
import reactCompiler from 'eslint-plugin-react-compiler'

/** @type {import('eslint').Linter.Config[]} */
const configs = [
  {
    plugins: {
      'react-compiler': reactCompiler,
    },
    rules: {
      'react-compiler/react-compiler': 'error',
    },
  },
]

export default configs
```

Reply:
@poteto @harish-sethuraman oh lol too slow (thanks for the merge!)

in case this is a desirable change, I can open a separate PR

Reply:
I'd prefer leaving it off since this is more about personal preference. We just want the minimal config in the docs.